src/devices/bus/multibus/isbc202.h

```diff
@@ -31,7 +31,7 @@ class isbc202_device : public cpu_device,
 	uint8_t io_r(address_space &space, offs_t offset);
 	void io_w(address_space &space, offs_t offset, uint8_t data);
 
-	DECLARE_WRITE_LINE_MEMBER(co_w);
+	void co_w(bool state);
```

Comment:
Why the change?  We still conventionally use `DECLARE_WRITE_LINE_MEMBER` for lines like this.